google/auth/transport/aiohttp_req.py

```diff
@@ -0,0 +1,297 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transport adapter for Async HTTP (aiohttp)."""
+
+from __future__ import absolute_import
+
+import asyncio
+import functools
+import logging
+
+
+import aiohttp
+import six
+
+
+from google.auth import exceptions
+from google.auth import transport
+from google.auth.transport import requests
+
+
+_OAUTH_SCOPES = [
```

Comment:
are these shared with the other transport? Can we extract them to a common import to avoid duplication?

Reply:
Fixed in the latest commit.